File: sdk/java/src/main/java/com/hjtpx/captcha/model/ConnectCaptchaResponse.java
package com.hjtpx.captcha.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class ConnectCaptchaResponse {
    @JsonProperty("session_id")
    private String sessionId;

    @JsonProperty("image_url")
    private String imageUrl;

    @JsonProperty("pairs")
    private List<PairItem> pairs;

    public static class PairItem {
        @JsonProperty("left")
        private int left;

        @JsonProperty("right")
        private int right;

        public int getLeft() {
            return left;
        }

        public void setLeft(int left) {
            this.left = left;
        }

        public int getRight() {
            return right;
        }

        public void setRight(int right) {
            this.right = right;
        }
    }

    public String getSessionId() {
        return sessionId;
    }

    public void setSessionId(String sessionId) {
        this.sessionId = sessionId;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public List<PairItem> getPairs() {
        return pairs;
    }

    public void setPairs(List<PairItem> pairs) {
        this.pairs = pairs;
    }
}

File: sdk/csharp/src/Hjtpx.Captcha.Sdk/Models/ConnectCaptchaResponse.cs
using System.Text.Json.Serialization;

namespace Hjtpx.Captcha.Models;

public class ConnectCaptchaResponse
{
    [JsonPropertyName("session_id")]
    public string SessionId { get; set; } = string.Empty;

    [JsonPropertyName("image_url")]
    public string ImageUrl { get; set; } = string.Empty;

    [JsonPropertyName("pairs")]
    public List<PairItem>? Pairs { get; set; }

    public class PairItem
    {
        [JsonPropertyName("left")]
        public int Left { get; set; }

        [JsonPropertyName("right")]
        public int Right { get; set; }
    }
}
